### PR TITLE
feat(Sicilia Vola): [IASV-23] Select beneficiary category screen

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR add the screen that allow the users to choose the category to which they belong.

## List of changes proposed in this pull request
- Add UI elements to the `SelectBeneficiaryCategoryScreen`
- Complete the `SvSelectBeneficiaryCategoryKoScreen`
- Modify the management of the workunit adding a navigation back if the workunit starts from 

https://user-images.githubusercontent.com/11773070/127887354-cb2f4da7-d5f8-43ca-8ddc-4d204e0556d9.mp4

## How to test
Starts the flow from the SiciliaVola service detail. Make sure you have the updated version of the master branch on dev-server .

### CODE REVIEW

File: ts/features/bonus/siciliaVola/screens/voucherGeneration/ko/SvSelectBeneficiaryCategoryKoScreen.tsx [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
+import { svGenerateVoucherCancel } from "../../../store/actions/voucherGeneration";
 
-const SvCheckIncomeKoScreen = () => (
-  <>
-    <View spacer={true} extralarge={true} />
-    <View spacer={true} extralarge={true} />
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const SvCheckIncomeKoScreen: React.FC<Props> = (props: Props) => (
```
Comment (Undermaken) [inline ts/features/bonus/siciliaVola/screens/voucherGeneration/ko/SvSelectBeneficiaryCategoryKoScreen.tsx:18]:
```suggestion
const SvCheckIncomeKoScreen: React.FC<Props> = (props: Props) => (
```
Could the screen name and its file match?
This is a little misleading

File: ts/features/bonus/siciliaVola/screens/voucherGeneration/SelectBeneficiaryCategoryScreen.tsx [hunk 2]
```diff
@@ -14,25 +15,64 @@ import {
...
+
+type BeneficiaryCategory = SvBeneficiaryCategory | "other";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
+const getCategoryBeneficiaryItems = (): ReadonlyArray<{
+  label: string;
+  id: BeneficiaryCategory;
+}> => [
```
Comment (Undermaken) [inline ts/features/bonus/siciliaVola/screens/voucherGeneration/SelectBeneficiaryCategoryScreen.tsx:38]:
```suggestion
const getCategoryBeneficiaryItems = (): ReadonlyArray<
  RadioItem<BeneficiaryCategory>
> => [
```
by exporting RadioItem you'll have a more compact type declaration

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/features/bonus/siciliaVola/screens/voucherGeneration/CheckStatusRouterScreen.tsx [hunk 3]
```diff
@@ -90,6 +92,7 @@ const CheckStatusRouterScreen = (props: Props): React.ReactElement => {
   );
 };
 const mapDispatchToProps = (dispatch: Dispatch) => ({
+  start: () => dispatch(svGenerateVoucherStart()),
   back: () => dispatch(svGenerateVoucherBack()),
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/features/bonus/siciliaVola/screens/voucherGeneration/CheckStatusRouterScreen.tsx:96]: Oh sorry, this is a copy-paste bug coming from the screen stub [PR](#3212 ).

Aligned all the KO components name in [9198089](https://github.com/pagopa/io-app/pull/3273/commits/9198089643bd48e55bca81c2e7a5cc2dd240c10e) and added the exit buttons in [b94ca91](https://github.com/pagopa/io-app/pull/3273/commits/b94ca9160e61de2a303ca7c314e9fe147acaa0de)

File: ts/features/bonus/siciliaVola/screens/voucherGeneration/ko/SvSelectBeneficiaryCategoryKoScreen.tsx (hunk 1 above)
Comment (debiff) [reply ts/features/bonus/siciliaVola/screens/voucherGeneration/ko/SvSelectBeneficiaryCategoryKoScreen.tsx]: Sure, related to the previous reporting.
Aligned in [9198089](https://github.com/pagopa/io-app/pull/3273/commits/9198089643bd48e55bca81c2e7a5cc2dd240c10e)

File: ts/features/bonus/siciliaVola/screens/voucherGeneration/SelectBeneficiaryCategoryScreen.tsx (hunk 2 above)
Comment (debiff) [reply ts/features/bonus/siciliaVola/screens/voucherGeneration/SelectBeneficiaryCategoryScreen.tsx]: Surely better! Thank you! 
Updated in [3bf3fb7](https://github.com/pagopa/io-app/pull/3273/commits/3bf3fb7533bed5cdb9e7e84f42b2713f377ea343)

**Review by Undermaken — APPROVED** [review]

**Pull request merged**